main/auth/linkedin.py

```diff
@@ -11,40 +12,48 @@
 
 from main import app
 
+
+def linkedin_compliance_fix(session):
```

Comment:
there is no need for this `linkedin_compliance_fix` with Authlib 0.14.3 now.

Reply:
Not needed, or is it breaking something?

As this PR would make it work in the first place, I would like it to be merged as-is; after that we can see if there are further improvements possible. Simply removing this line (or code block) will not work... it is used somewhere else.

Alternatively, if this is causing a problem (with Authlib to keep this PR as-is): another PR using the newer Authlib that re-enables Linkedin as a login method is an option (and then this PR could be closed without merging).

Reply:
@mdxs this `linkedin_compliance_fix` won't break anything.

Reply:
Thanks, then I will leave it as-is for now; then see if we can indeed simply it later.

Reply:
Actually, just implemented suggestion of @lepture (to drop the linkedin_compliance_fix as it is not needed with Authlib 0.14.3 anymore). Squashed the commit log, to simplify merge.